modules/fundamental/src/license/service/mod.rs

```diff
@@ -206,4 +210,45 @@ impl LicenseService {
         }
         Ok(None)
     }
+
+    pub async fn get_all_license_info<C: ConnectionTrait>(
+        &self,
+        sbom_id: Id,
+        connection: &C,
+    ) -> Result<HashSet<String>, Error> {
+        if let Some(uuid) = sbom_id.try_as_uid() {
+            let stmt = Statement::from_sql_and_values(
+                connection.get_database_backend(),
+                r#"
+        (
+            SELECT DISTINCT unnest(l.spdx_licenses) as license_id
+            FROM sbom_package_license spl
+            JOIN license l ON spl.license_id = l.id
+            WHERE spl.sbom_id = $1
+            AND l.spdx_licenses IS NOT NULL
+        )
+        UNION
+        (
+            SELECT DISTINCT license_id
+            FROM licensing_infos
+            WHERE sbom_id = $1
+        )
```

Comment:
When executing this, I get a lot of `LicenseRef-1` entries. These look like internal references, which have no meaning to the user. Shouldn't we instead report the actual license this is referring to?

Reply:
This is to provide data for the package's license filter. From the current requirements, this is necessary.